app/controllers/registers_controller.rb

```diff
@@ -7,14 +7,25 @@ class RegistersController < ApplicationController
 
   def index
     @search_term = search_term
-    @registers = Register.available
-                         .in_beta
-                         .search_registers(@search_term)
 
     # Redirect legacy URL to ensure we don't break anyone
     if params[:phase] == 'in progress'
       redirect_to registers_in_progress_path
     end
+
+    if params[:showby] == 'name' || @search_term.present?
```

Comment:
can we use `show_by` as it is more consistent with our other params?

Reply:
Done.